Source/ACE/Managers/GuidManager.cs

```diff
@@ -1,46 +1,66 @@
-﻿using System;
-using System.Collections;
-
-namespace ACE.Managers
+﻿namespace ACE.Managers
 {
     /// <summary>
     /// Used to assign global guids and ensure they are unique to server.
     /// todo:// use and reuse .. keep track of who using what and release ids..
     /// </summary>
     public static class GuidManager
     {
-        // thanks pea for good ranges!
-
-        // players
-        private const uint playerMin = 0x50000001;
-        private const uint playerMax = 0x59999999;
-        private static uint player   = 0x50000001;
+        // Running server is guid master - database only read as startup to get current max per range.
+        // weenie class templates Max 65,535 - took Turbine 17 years to get to 10K
+        // these will be added by developers and not in game.
+        // Nothing in this range is persisted by the game.   Only developers or content creators can create them to be persisted.
+        // Fragmentation: None
+        private const uint weenieMin = 0x00000001;
+        private const uint weenieMax = 0x0000FFFF;
+        // At Server startup read current max from the DB
+        private static uint weenie   = 0x00001344;
```

Comment:
I have no objections to the hard-coding the weenie ID ranges.  If it makes you feel better, we can even put contraints on the database to mirror these values.  IIRC, these are hard coded in the client as well.

The "current max" does seem a bit... unnecessary.  Ctrl+F suggests nothing is calling NewWeenieGuid( ) - what is it intended for?